core/core_modules/issue.js

```diff
@@ -22,97 +22,99 @@ catch (e) {
     console.debug('Failed to load config file for core module issue');
 }
 
-var submitRequest = function(title, description, debugLevel, callback, waitCallback) {
-    waitCallback();
+var ifStringNotEmpty = function(str) {
+    return str && str.trim(). length !== 0;
+},
 
-    var data = {title: title},
-        debugData = null;
+    submitRequest = function(title, description, debugLevel, callback, waitCallback) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 17.](https://www.codacy.com/app/theprojectshop/Kassy/file/2589031844/issues/source?bid=3282987&fileBranchId=3291254#l29)